vroom/main.vroom

```diff
@@ -18,7 +18,30 @@ stub that out for vroom.
   :endfunction
   :call maktaba#test#Override('repeat#set', 'FakeRepeat')
 
-We'll also stub it out to not care whether certain executables are installed on
+
+Before we stub out availability checks, test :FormatCode for unavailable plugins
```

Comment:
Actually, I have spent the last hour or so trying to make that work.  I clearly don't understand something about vim's evaluation of globals or something.  Do you mind taking a crack at it?